blaze-core/src/test/scala/org/http4s/blaze/util/EntityBodyWriterSpec.scala

```diff
@@ -232,34 +228,30 @@ class EntityBodyWriterSpec extends Http4sSpec {
 
     "write a deflated resource" in {
       val p = resource through deflate()
-      p.runLog.map(_.toArray) must returnValue(DumpingWriter.dump(p))
+      p.runLog.map(_.toArray) must returnValue(DumpingWriter.dump(resource through deflate()))
```

Comment:
No, I didn't try changing anything on master. But no matter what version I try or how I arrange things, the second time that stream is run, it fails in the deflate step.

Since it's both run directly in the test and by the `DumpingWriter`, I don't see how this test worked to begin with.

See https://scastie.scala-lang.org/aeons/igTcf7D9SOmXv94CzXZRUA/1